### PR TITLE
Replace logic that strips out HTML tags with API calls for plaintext values.

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
This PR addresses some technical debt by removing logic that strips out HTML tags from section and page titles in the front end and replace it with a query for a field on the GraphQL API that retrieves plaintext representations (using the `format: Plaintext` field argument).

The aim going forward is for the API to be the single source of truth for the data and to remove and to keep the logic for stripping out HTML tags in one place. 

### How to review 
Plaintext titles are displayed in the navigation, move page/section modals and the piping menu. A visual inspection should be carried out to ensure that titles are displayed correctly.

All tests and checks should pass and the changes should look sensible.

### CODE REVIEW

**Comment by adammahmood** [comment]
LGTM.  However, looks like Cypress tests are failing.

File: src/components/RichTextEditor/Piping.graphql [hunk 1]
```diff
@@ -4,10 +4,12 @@ query GetQuestionnaire_Piping($id: ID!) {
     sections {
       id
       title
+      plaintextTitle: title(format: Plaintext)
```
Comment (SamGodwin2) [inline src/components/RichTextEditor/Piping.graphql]: Do we still need to request title here or can we remove it?

File: src/graphql/getQuestionnaire.graphql [hunk 2]
```diff
@@ -6,6 +6,7 @@ query GetQuestionnaire($id: ID!) {
     sections {
       id
       title
+      plaintextTitle: title(format: Plaintext)
```
Comment (SamGodwin2) [inline src/graphql/getQuestionnaire.graphql]: Same again here

File: src/components/RichTextEditor/Piping.graphql (hunk 1 above)
Comment (samiwel) [reply src/components/RichTextEditor/Piping.graphql]: Good shout

**Pull request merged**